doc/cprover-manual/contracts-decreases.md

```diff
@@ -1,3 +1,179 @@
-# Decreases Clause
+# Decreases Clauses
 
-TODO: Document `__CPROVER_decreases`
+A _decreases_ clause specifies a measure that must strictly decrease at every iteration of a loop.
+By demonstrating that the measure
+1. is bounded from below, and
+2. strictly decreases at each iteration,
+we can prove termination of loops.
+This is because the measure must eventually hit the lower bound
```

Comment:
You're appending the last paragraph in `2.`. Add an empty line between them (or write them as a single paragraph).

Reply:
Good catch! Thanks